Mathlib/Data/Finset/Card.lean

```diff
@@ -120,6 +120,26 @@ theorem card_insert_le (a : α) (s : Finset α) : card (insert a s) ≤ s.card +
   · rw [card_insert_of_not_mem h]
 #align finset.card_insert_le Finset.card_insert_le
 
+section
+
+variable {a b c d e f : α}
+
+theorem card_le_two : card {a, b} ≤ 2 := card_insert_le _ _
```

Comment:
Maybe we also want an alias called `Finset.card_pair` (since we have `Multiset.card_pair)`.

Reply:
There is already `Finset.card_doubleton`, which assumes `a \ne b`. Maybe it should be renamed... @YaelDillies what do you think?

Reply:
Yes, `card_doubleton` should be called `card_pair`. I complained when Kyle added that lemma, but I wasn't heard.

Reply:
See #9379